doc/releases/octopus.rst

```diff
@@ -1,3 +1,17 @@
+v15.2.6 Octopus
+===============
+
+This is the 6th backport release in the Octopus series. This releases fixes a
+security flaw affecting Messenger V2 for Octopus & Nautilus. We recommend users
```

Comment:
Similarly, "flaw affecting both Messenger v1 and Messenger v2, among ...".

In octopus and master (but not in nautilus), both msgr1 and msgr2 are affected.